### ISSUE TITLE
Add an option to remove the "you are opening a codesandbox preview" warning on the preview page

### ISSUE THREAD

**Issue by anton-series** [codesandbox/codesandbox-sdk#5]
Please add the ability to remove the warning screen that appears before the preview, or mention in the docs that the screen is non-removable. 

We are showing temporary apps running in the sandbox for our users and they are confused when they see this screen.

<img width="323" alt="Screenshot 2024-12-17 at 10 51 05 AM" src="https://github.com/user-attachments/assets/bfbc596b-541a-4670-bad9-eec49e08e57d" />

**Comment by CompuIves** [comment]
Hey! Yes, this is an important thing we should fix asap. I expect we'll be able to get rid of this in first week of January.

For an intermediate solution, we don't show this interstitial for pro users (with the idea that users who create phishing pages are not pro).

**Comment by christianalfoni** [comment]
Yeah, we need to keep this for free users to avoid phishing issues.
